pkgs/tools/graphics/wgpu/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, rustPlatform, fetchFromGitHub, pkg-config, makeWrapper, vulkan-loader }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "wgpu";
+  version = "0.10.0";
+
+  src = fetchFromGitHub {
+    owner = "gfx-rs";
+    repo = pname;
+    rev = "9da5c1d3a026c275feb57606b8c8d61f82b43386";
+    sha256 = "sha256-DcIMP06tlMxI16jqpKqei32FY8h7z41Nvygap2MQC8A=";
```

Comment:
```suggestion
    rev = "v${version}";
    sha256 = "...";
```

Reply:
Unfortunately upstream doesn't use git tags, so we need to use the commit id here. I already [asked them to tag their releases](https://github.com/gfx-rs/wgpu/issues/1638#issuecomment-933026031).

Reply:
my bad